h2o-genmodel/src/main/java/hex/genmodel/algos/tree/SharedTreeMojoModel.java

```diff
@@ -80,7 +80,7 @@ public final int getNTreesPerGroup() {
    * Note: this function is also used from the `hex.tree.CompressedTree` class in `h2o-algos` project.
    */
   @SuppressWarnings("ConstantConditions")  // Complains that the code is too complex. Well duh!
-    public static double scoreTree(byte[] tree, double[] row, int nclasses, boolean computeLeafAssignment, String[][] domains) {
+    public static double scoreTree(byte[] tree, double[] row, boolean computeLeafAssignment, String[][] domains) {
```

Comment:
A good practice is to keep backward compatibility method that has the parameter and delegates to the simpler version + mark the old one deprecated.

Reply:
I just supposed that this method is not effectively facing users, but you are right, and what you say is definitely safer and makes no harm anyway, thx